android/app/src/bt/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
   <!-- Shown on app info, app switcher, etc -->
-  <string name="app_name" translatable="false">PathCheck MN</string>
+  <string name="app_name" translatable="false">PathCheck BT</string>
```

Comment:
This was updated in a previous commit and we think by default should just be BT right?

Reply:
yeah, it likely should be read from a config file at somepoint to avoid this issue.

Reply:
thats what this pr does!